index.html

```diff
@@ -11,26 +11,29 @@
 
 <body>
   <section class="Container-1">
-    <main class="header-mobile">
-
+    
       <header class="menu1">
         <nav class="navigation">
           <a href="#" class="logo">My Logo</a>
-          <img src="media/Union.png" alt="" width="30px">
-          <ul>
-            <li>
-              <a href="">Portfolio</a>
+          <ul class="navigation-two">
+            <li class="nav">
+              <a href="#">Portfolio</a>
             </li>
-            <li>
-              <a href="">About</a>
+            <li class="nav">
+              <a href="#">About</a>
             </li>
-            <li>
-              <a href="">Contact</a>
+            <li class="nav">
+              <a href="#">Contact</a>
             </li>
           </ul>
```

Comment:
- [ ] Your mobile menu is great! 💯 However, there's an extra requirement that is not yet met:

![image](https://user-images.githubusercontent.com/77069011/202289810-d7650352-3cc2-45b7-9953-194cf88642c2.png)

The main functionality of your mobile menu is to make navigation ⛵ through your page easy for users! Please, consider implementing this. 🤓 🛠️ 

💡 _Hint: You could use id attributes in all your sections, and, since you're using `<a>` tags, you could replace the `href="#"`, for `href="#[SECTION ID]"` when the user clicks the link, is going to get the user to the HTML element that matches the id._

Example:

```html
<a href="#portfolio">Portfolio</a>

<section id="portfolio"> 
  ...
</section>
```